### PR TITLE
Team5 maxhp

### PR DESCRIPTION
**Pull request by tchiko7**
# Summary

added function to get max  hp and percentage hp

## Additional Information

also changed existing hp getters to call the percentageHP function

### CODE REVIEW

**Review by hussain2603 — APPROVED** [review]

File: pkg/agents/team5/agent1/agent.go [hunk 1]
```diff
@@ -49,6 +49,14 @@ func New(baseAgent *infra.Base) (infra.Agent, error) {
 	}, nil
 }
 
+func MaxHP(a *CustomAgent5) int {
+	return a.HealthInfo().MaxHP
+}
+
+func PercentageHP(a *CustomAgent5) float64 {
+	return float64(a.HP()) / float64(MaxHP(a))
+}
```
Comment (jzzheng22) [inline pkg/agents/team5/agent1/agent.go:58]: I'd prefer to keep this as an int

File: pkg/agents/team5/agent1/agent.go [hunk 2]
```diff
@@ -49,6 +49,14 @@ func New(baseAgent *infra.Base) (infra.Agent, error) {
 	}, nil
 }
 
+func MaxHP(a *CustomAgent5) int {
+	return a.HealthInfo().MaxHP
+}
```
Comment (jzzheng22) [inline pkg/agents/team5/agent1/agent.go:54]: Why not just call `a.HealthInfo().MaxHP` directly??

**Review by jzzheng22 — CHANGES_REQUESTED** [review]

File: pkg/agents/team5/agent1/agent.go (hunk 2 above)
Comment (tchiko7) [reply pkg/agents/team5/agent1/agent.go:54]: i thought code would be easier to read, i can change if you like

Comment (jzzheng22) [reply pkg/agents/team5/agent1/agent.go:54]: Ehhh we only use it in one place so far, I think we don't need the function - if we use a lot more and it gets annoying, then maybe yeah.

**Review by jzzheng22 — APPROVED** [review]

**Comment by jzzheng22** [comment]
Thanks Osman

**Pull request merged**